source/fundamentals/logging.txt

```diff
@@ -20,6 +20,7 @@ Logging
 
   - :doc:`Command Monitoring </fundamentals/monitoring/command-monitoring>`
   - :doc:`Cluster Monitoring </fundamentals/monitoring/cluster-monitoring>`
+  - :doc:`Connection Pool Monitoring </fundamentals/monitoring/connection-monitoring>`
```

Comment:
Suggestion:
I see that you're following what's already there, but the team decided that we should use refs whenever possible instead of doc. No need to change the Command and Clustering monitoring links unless you prefer to do so, nor to ticket any of the work to convert all doc links to ref ones.

Reply:
Updated all to refs.